### ISSUE TITLE
--scale

### ISSUE THREAD

**Issue by UnixJunkie** [UnixJunkie/svmwrap#4]
- learn from the training set scaling parameters to put all feature values inside [0:1]
- apply them on the test set

**Comment by UnixJunkie** [comment]
--iwn was repaired, in the mean time

**Comment by UnixJunkie** [comment]
done; but currently not supported in production (the scaling params would need to be stored along with the trained model)
